CONTRIBUTING.md

```diff
@@ -148,3 +148,15 @@ The pacmak code generator should be implemented under
 
 The [Python](./packages/jsii-pacmak/lib/targets/python.ts) target is a good
 example to work from.
+
+## Releasing
+### The `jsii/superchain` Docker image
+
+Upon merging new changes to the `master` branch, the `jsii/superchain:nightly`
+image will be released by TravisCI after a last validation build.
+
+Upon making a new `jsii` release (creating the version tag), the
```

Comment:
A couple of lines about how to create the version tag would be useful here. If it's already elsewhere, link to there.

Reply:
Ah! I see the ambiguity here - "tag" could mean "docker image tag" or "git tag" (and I mean the later here). I've disambiguated this, so hopefully this makes more sense now!